packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -8531,6 +8531,33 @@ describe('plugin-meetings', () => {
           });
         });
       });
+
+      describe('#brb', () => {
```

Comment:
Haven't written an SDK unit test in a while... just wondering if it would be more appropriate to put these tests under the "Public Api Contract" describe block? It would test the multistream vs non-multistream case. There's also "locusMediaRequestStub" that you might be able to reuse there.